sox.features.js

```diff
@@ -646,109 +635,122 @@
     sortByBountyAmount: function() {
       // Description: For adding some buttons to sort bounty's by size
 
+      const sortButton = `<button class="s-btn s-btn__muted s-btn__outlined s-btn__sm s-btn__dropdown" role="button" data-controller="s-popover"
+                                  data-action="s-popover#toggle" data-target="se-uql.bountyAmount" aria-haspopup="true"
+                                  aria-expanded="false" aria-controls="sox-bounty-popover">SOX: sort by bounty</button>`
+      const sortPopover = `<div class="s-popover z-dropdown ws2" id="sox-bounty-popover" data-target="se-uql.bountyAmount"
+                                style="margin: 0px;" data-popper-placement="bottom">
+                               <div class="s-popover--arrow" style="position: absolute; left: 0px; transform: translate(99px, 0px);"></div>
+                               <ul class="list-reset mtn8 mbn8 js-uql-navigation">
+                                   <li class="uql-item my0"><a id="largestFirst" class="mln12 mrn12 px12 py6 fl1 s-block-link">Largest first</a></li>
+                                   <li class="uql-item my0"><a id="smallestFirst" class="mln12 mrn12 px12 py6 fl1 s-block-link">Smallest first</a></li>
+                               </ul>
+                           </div>`
+
       // Do nothing unless there is at least one bounty on the page
-      if (!document.getElementsByClassName('bounty-indicator').length) return;
+      if (!document.getElementsByClassName('bounty-indicator')) return;
```

Comment:
I think we need to keep the `.length` because an empty `HTMLCollection` seems to be truthy